editor/actions.js

```diff
@@ -56,11 +56,9 @@ export function editPost( edits ) {
 	};
 }
 
-export function savePost( postId, edits ) {
+export function savePost() {
```

Comment:
We may want to provide a way to explicitly pass the edits we want to persist. I'm wondering if the "Revert to draft" button should save all the current edits or only the status (could be resolved in the "Revert to draft" PR)